R/release.R

```diff
@@ -403,8 +403,25 @@ get_release_news <- function(SHA = gert::git_info(repo = git_repo())$commit,
   }
 }
 
-cran_version <- function(package = project_name(),
-                         available = utils::available.packages()) {
+cran_version <- function(package = project_name(), available = NULL) {
+
+  if (!curl::has_internet()) {
+    return(NULL)
+  }
+
+  if (is.null(available)) {
+  # Guard against CRAN mirror being unset
+    available <- try(
```

Comment:
We sort of have a policy about never using `try()`, but instead using `tryCatch()`.

https://github.com/tidyverse/design/issues/85

Can you rewrite this using `tryCatch()`?

Reply:
Ok! I had looked in the design guide about this, but not in the issues ;)

Reply:
Well, this is more folklore at this point. It needs to be written up properly.